models/setting.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require('../mongoose');
+const uuid = require('uuid');
+const Schema = mongoose.Schema;
+
+const SettingSchema = new Schema({
+  anonymous_users: {type: Boolean, default: false},
```

Comment:
can we divide somewhow the settings and make comments on them? They are a lot and a big block of code.